src/sync.js

```diff
@@ -72,11 +73,36 @@ async function processBatch(txs) {
         break;
       case "custom_json":
         if (payload.id === "follow") {
-          const { follower, following, what } = JSON.parse(payload.json);
-          if (what.includes("blog")) {
-            await addFollow(timestamp, follower, following);
-          } else if (what.includes("ignore") || what.length === 0) {
-            await removeFollow(timestamp, follower, following);
+          const json = JSON.parse(payload.json);
+          switch (json[0]) {
```

Comment:
I would rather check if `json` is an array or not. Right now if there happens to be payload like this:
`['whatever', { }]` it would think it's a follow, which it's not.

Reply:
If payload is `['whatever'], {}` it will endup here https://github.com/busyorg/busy-db/blob/fc119acfa6594efd57e87713fe45756606aa7f96/src/sync.js#L114 , it will not be considered as a follow.

Reply:
I mean transaction with id of `custom_json`. and `json` equals to `['whatever', { }]`.

Reply:
Like this https://jsfiddle.net/b835sje4/4/ ?? Not sure to understand. With `switch (json[0])`, if `json[0]` is undefined it will goes into `default` case

Reply:
I missed this check, but my biggest concern is that we rely on `obj[0]` returning `undefined` as a control of flow.

I would rather explicitly check to see if it's an array of object and react accordingly.

Reply:
I will workout something

Reply:
It should be good now